basics/getting_started.md

```diff
@@ -3,18 +3,22 @@ layout: contribute
 title: Getting Started
 ---
 
-# Setting up your coding environment
+# Getting started
 
-For now we have support for IntelliJ. We don't have IDE support for other
-languages in Bazel right now.
+This page helps you set up your coding environment, describes creating an
+IntelliJ project and takes you through compiling and debugging your project.
+Finally, you can read about Bazel's source code, how to navigate through it, and
```

Comment:
After you set up your environment, there is a quick overview of the structure of Bazel's code base, how to search and navigate the code, and how to monitor your builds with Bazel's continuous integration system.

Reply:
Done.